packages/browser-utils/src/metrics/web-vitals/getLCP.ts

```diff
@@ -82,10 +83,12 @@ export const onLCP = (onReport: LCPReportCallback, opts: ReportOpts = {}) => {
       // stops LCP observation, it's unreliable since it can be programmatically
       // generated. See: https://github.com/GoogleChrome/web-vitals/issues/75
       ['keydown', 'click'].forEach(type => {
-        // Wrap in a setTimeout so the callback is run in a separate task
-        // to avoid extending the keyboard/click handler to reduce INP impact
-        // https://github.com/GoogleChrome/web-vitals/issues/383
-        addEventListener(type, () => setTimeout(stopListening, 0), true);
+        if (WINDOW.document) {
+          // Wrap in a setTimeout so the callback is run in a separate task
+          // to avoid extending the keyboard/click handler to reduce INP impact
+          // https://github.com/GoogleChrome/web-vitals/issues/383
+          WINDOW.document.addEventListener(type, () => setTimeout(stopListening, 0), true);
```

Comment:
l: for bundle size reasons: Is it necessary to explicitly call `addEventListener` on `WINDOW.document` if we have the guard above?